examples/connext_dds/detect_samples_dropped/java/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+  id 'com.github.rticommunity.connext-dds-build-example'
+}
+
+ConfigureExample {
+  idlFile = 'DroppedSamplesExample.idl'
+  codegenExtraArgs = ['-update', 'examplefiles']
```

Comment:
I think that the 'update' flag will replace the publisher and subscriber, isn't it? (This comment applies to other examples as well)

Reply:
No, it will not replace them.